.mk/local.mk

```diff
@@ -0,0 +1,39 @@
+##@ Local (Kind)
+
+.PHONY: create-kind-cluster
+create-kind-cluster: $(KIND) ## Create kind cluster
+	-$(KIND) create cluster --config config/kubernetes/kind.config.yaml
+	kubectl cluster-info --context kind-kind
+
+.PHONY: delete-kind-cluster
+delete-kind-cluster: $(KIND) ## Delete kind cluster
+	$(KIND) delete cluster
+
+.PHONY: local-deploy
+local-deploy: create-kind-cluster deploy-all  ## Local deploy (kind, loki, grafana and example-cr)
```

Comment:
I ran `make local-deploy` expecting it would deploy everything, especially as I see a `deploy-all` target here, but it doesn't seem to deploy (run) the operator, is it as intended?

I get that it's hard to name everything correctly as there are many different targets.

Maybe add some doc in the README for these new targets (not super-detailed, but in the main lines...)